src/components/LabelWarning.vue

```diff
@@ -0,0 +1,163 @@
+<script>
+import { mapGetters } from 'vuex'
+export default {
+  props: {
+    flow: {
+      type: Object,
+      default: () => {}
+    },
+    flowGroup: {
+      type: Object,
+      default: () => {}
+    },
+    iconSize: {
+      type: String,
+      required: false,
+      default: null
+    },
+    location: {
+      type: String,
+      default: ''
+    }
+  },
+  data() {
+    return {
+      infoMessage: ''
+    }
+  },
+  computed: {
+    ...mapGetters('tenant', ['tenant']),
+    ...mapGetters('agent', ['agents']),
+    agentOrLabel() {
+      if (!this.agents || !this.agents.length) return 'Agent Problem'
+      if (!this.labelsAlign) return 'Label Problem'
+      return 'Flow and Agent Labels'
+    },
+    agentLabels() {
+      return this.agents?.reduce((accumulator, agent) => {
+        accumulator.push(agent.labels)
+        return accumulator
+      }, [])
+    },
+    flowLabels() {
+      return this.flowGroup?.labels || this.flow?.environment?.labels
+    },
+    docsName() {
+      if (!this.agents?.length) return 'agents'
+      return 'Labels and Flow Affinity'
+    },
+    docsLink() {
+      if (!this.agents?.length)
+        return 'https://docs.prefect.io/orchestration/agents/overview.html'
+      return 'https://docs.prefect.io/orchestration/agents/overview.html#flow-affinity-labels'
+    },
+    labelsAlign() {
+      if (!this.agents?.length) {
+        this.labelMessage(
+          'You have no live Agents - scheduled flow runs will not be submitted for execution and will display as "Late".'
```

Comment:
```suggestion
          'You have no live Agents - scheduled flow runs will not be submitted for execution and might display as "Late".'
```

Super nit-picky, but there could be situations where a Flow is scheduled far in the future and doesn't have active Agents polling